workflow/5_adaptation/adaptation_options_costs.py

```diff
@@ -0,0 +1,381 @@
+    road_costs = adapt_costs[adapt_costs["asset_description"] == "roads"]
+    roads_df = []
+    for rc in road_costs.itertuples():
+        if rc.asset_details == "Roads-2L":
+            df = asset_df[asset_df["lanes"] == 2]
+            df["lane_factor"] = 1
+        elif rc.asset_details == "Roads-4L":
+            df = asset_df[asset_df["lanes"] != 2]
+            df["lane_factor"] = df["lanes"] / 4.0
+        else:
+            df = asset_df.copy()
+            df["lane_factor"] = 1
+
+        for column in [
+            "asset_dimensions",
+            "change_parameter",
+            "currency_conversion",
+            "asset_dimension_coversion",
+            "adaptation_option",
+            "option_unit_cost",
+            "initial_investment_cost_per_unit",
+            "periodic_maintenance_cost_per_unit",
+            "routine_maintenance_cost_per_unit",
+            "periodic_maintenance_intervals_years",
+            "routine_maintenance_intervals_years",
+        ]:
+            if column == "asset_dimension_coversion":
+                df[column] = getattr(rc, column) * df["lane_factor"]
+            else:
+                df[column] = getattr(rc, column)
+        df = get_adaptation_options_costs(df, asset_id)
+        roads_df.append(df)
+
+    roads_df = pd.concat(roads_df, axis=0, ignore_index=True)
+    return roads_df
+
+
+@click.command()
+@click.version_option("1.0")
+@click.option(
+    "--asset-data",
+    "-a",
+    required=True,
+    type=click.Path(
+        exists=True,
+        dir_okay=False,
+        file_okay=True,
+        readable=True
+    ),
+    help="Path to the asset layer data",
+)
+@click.option(
+    "--asset-file",
+    "-b",
+    required=True,
+    type=click.Path(
+        exists=True,
+        dir_okay=False,
+        file_okay=True,
+        readable=True
+    ),
+    help="Path to asset gpkg file",
+)
+@click.option(
+    "--cost-file",
+    "-c",
+    required=True,
+    type=click.Path(
+        exists=True,
+        dir_okay=False,
+        file_okay=True,
+        readable=True
+    ),
+    help="Path to adaptation cost data",
+)
+@click.option(
+    "--hazard-label",
+    "-h",
+    required=True,
+    help="hazard label",
+)
+@click.option(
+    "--asset-gpkg",
+    "-g",
+    required=True,
+    help="asset_gpkg value in the network CSV",
+)
+@click.option(
+    "--asset-layer",
+    "-l",
+    required=True,
+    help="asset_layer value in the network CSV",
+)
+@click.option(
+    "--output-dir",
+    "-o",
+    required=True,
+    type=click.Path(
+        exists=True,
+        dir_okay=True,
+        file_okay=False,
+        readable=True
+    ),
+    help="Path to the output directory",
+)
+def adaptation_options_costs(
+    asset_data,
+    asset_file,
+    cost_file,
+    hazard_label,
+    asset_gpkg,
+    asset_layer,
+    output_dir,
+):
+    epsg_jamaica = 3448
+    baseline_year = 2019
+    projection_end_year = 2100
+    discounting_rate = 10
```

Comment:
If we're factoring these out in #47, we should do it here too. Perhaps they should all live in the top level `config.json` with slightly more descriptive keys, e.g. "adaptation_options_baseline_year", "adaptation_options_end_year", "adaptation_option_discount_rate".

Reply:
Or nest them under a parent "adaptation_options" key